cpu/load_balancer.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Samir A Mulani <samir@linux.vnet.ibm.com>
+
+import multiprocessing
+import os
+import re
+import time
+import pandas as pd
+from avocado import Test
+from avocado.utils import process, cpu, distro
+from avocado.utils.software_manager.manager import SoftwareManager
+
+totalcpus = int(multiprocessing.cpu_count()) - 1
+file_path = "/tmp/mpstat.log"
+online_cpu_check = "/sys/devices/system/cpu/"
+cpu_payload_data = []
+if os.path.exists(file_path):
+    os.remove(file_path)
```

Comment:
The above part of the code can go to setup and they can be class variable.. any reason to have this global ?

Reply:
> The above part of the code can go to setup and they can be class variable.. any reason to have this global ?
The reason behind this is to make the scope of the above code visible throughout all the functions, but adding it to the setup will make sense.

Reply:
@SamirMulani class variable have scope in all function, so you access them from any function